opencog/atoms/base/Handle.cc

```diff
@@ -80,6 +80,21 @@ bool Handle::atoms_less(const Atom* pa, const Atom* pb)
     return a < b;
 }
 
+bool Handle::content_based_atoms_less(const Atom* a, const Atom* b)
+{
+    if (a == b) return false;
+    if (nullptr == a) return true;
+    if (nullptr == b) return false;
+    UUID ua = a->getUUID();
+    UUID ub = b->getUUID();
```

Comment:
OK, I see that's the part that made you assume I have abused UUID, but this is a just a copy/paste of atoms_less. Sure I could have striven to not to copy/paste anything but I didn't want to refactor anything either so there it went.